src/main/java/net/snowflake/client/jdbc/cloud/storage/StageInfo.java

```diff
@@ -155,6 +156,14 @@ public boolean getIsClientSideEncrypted() {
     return isClientSideEncrypted;
   }
 
+  public void setUseS3RegionalUrl(boolean useS3RegionalUrl) {
```

Comment:
Can take out the "is" so that the variable name makes more sense as a boolean. `useS3RegionalUrl` instead of `isUsesS3RegionalUrl`

Reply:
Done